src/utils/statistics.C

```diff
@@ -253,12 +267,12 @@ void StatisticsVector<T>::histogram(std::vector<dof_id_type> & bin_members,
                                                static_cast<dof_id_type>(0),
                                                std::plus<dof_id_type>());
 
-  if (n != n_binned)
+  if (n-unbinned != n_binned)
     {
       libMesh::out << "Warning: The number of binned entries, n_binned="
                    << n_binned
                    << ", did not match the total number of entries, n="
```

Comment:
It might be more accurate for this warning to say "total number of *binnable* entries", since now we don't count the inactive ones.